gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -197,7 +197,11 @@ impl ClnRpcService {
             ))
         } else {
             Err(ClnExtensionError::Error(anyhow!(
-                "Failed to start cln plugin"
+                "
+                    Failed to start cln plugin - another instance of lightningd may already be running.
```

Comment:
Is running 2 lightningd nodes the only possible way this error message can be hit?

Reply:
It might be better to just print out the actual error message.  But it doesn't seem like we get one? Just a `None`?

Reply:
I actually don't see why running two CLNs should lead to that error, but apparently that's what @Maan2003 observed, so seems like worth mentioning in that message.

Reply:
I was incorrectly spawning two cln nodes with same commands (same datadir) concurrently.